RailwayCo/Assets/Scripts/Unity/Main/Right Panel/CargoDetailButton.cs

```diff
@@ -36,9 +39,23 @@ private void Awake()
 
     private void OnButtonClicked()
     {
-        // Buttion functionality should only be available when the cargo is in the station with a train inside.
+        // Button functionality should only be available when the cargo is in the station with a train inside.
         if (_currentTrainGUID == Guid.Empty || _currentStationGUID == Guid.Empty) return;
-        _logicMgr.ShiftCargoOnButtonClick(this.gameObject, _cargo, _currentTrainGUID, _currentStationGUID);
+        if (!_logicMgr.ShiftCargoOnButtonClick(this.gameObject, _cargo, _currentTrainGUID, _currentStationGUID))
+        {
+            string eventType = "";
+            CargoAssociation cargoAssoc = _cargo.CargoAssoc;
+            if (cargoAssoc == CargoAssociation.STATION || cargoAssoc == CargoAssociation.YARD)
+                eventType = "Train capacity is full";
+            else if (cargoAssoc == CargoAssociation.TRAIN)
+                eventType = "Yard capacity is full";
+            TooltipManager.Show(eventType, "Error");
+        }
+    }
+
+    public void OnPointerExit(PointerEventData eventData)
```

Comment:
private this

Reply:
Unable to do so. On changing it to private, it will no longer be implementing the `IPointerExitHandler` interface.